force-app/main/default/permissionsets/NPSP_Donation_History_Community_User_managed.permissionset-meta.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Check with @jencrabtree on the naming of this permission set. I don't think we need to include the word "managed" in this perm set.

Reply:
It has been decided the Permission Set will be removed from this PR